src/services/base.service.js

```diff
@@ -215,6 +216,7 @@ const baseUpdate = async (args, { req }, resourceType) => {
   if (data.id !== args.id) {
     throw new BadRequestError('Argument id must match request body id for PUT request');
   }
+  data['meta'] = { lastUpdated: getCurrentInstant(), ...data['meta'] };
```

Comment:
See above comment about overwriting

Reply:
Fixed line 94 and 219, added comments about fix